gen/builders/state_tracker.go

```diff
@@ -62,12 +186,10 @@ func (st *StateTracker) Fork(root cid.Cid) *StateTracker {
 		panic(err)
 	}
 
-	return &StateTracker{
-		Stores:    st.Stores,
-		StateTree: tree,
-		Driver:    st.Driver,
-		CurrRoot:  root,
-	}
+	cpy := *st
+	cpy.StateTree = tree
+	cpy.CurrRoot = root
+	return &cpy
```

Comment:
Now that there are more fields in `StateTracker`, taking a full copy and overwriting specific fields is more reliable.